packages/generator-electrode-component/app/templates/_gulpfile.js

```diff
@@ -18,6 +18,9 @@ const tasks = {
   "generate": ["generate-metadata", "generate-documentation"],
   "generate-documentation": () => exec(`electrode-docgen --package ./package.json --src ./src --markdown components.md`),
   "generate-metadata": () => exec(`electrode-docgen --package ./package.json --src ./src --metadata components.json`),
+<% if (quoteType === "'") { %>
+  'archetype:lint-server': () => exec(`eslint --color -c ./.eslintrc config/karma config/webpack demo-server`),
```

Comment:
overwriting the task is ok, but I don't remember why these tasks are not in the component archetype.